### PR TITLE
Add actual lesson to seeds

### PR DESCRIPTION
**Pull request by peteryates**
### Context

All of our data is currently placeholder, mostly borrowed from Wikipedia.

### Changes proposed in this pull request 👨‍🏫

Try adding some _actual_ lesson content. Text provided by @Bethlt4, icons by @pripley-ea. Some minor code tweaks will be needed along the way.

### Guidance to review

Does it look ok? Does the [provided content](https://dfedigital.atlassian.net/wiki/spaces/CMB/pages/1372258318/Lesson+1+Contenders+for+the+throne) fit the structure properly?

### CODE REVIEW

**Comment by peteryates** [comment]
Preview of the knowledge tab:

Screenshot_2020-02-28 Curriculum Materials

**Comment by peteryates** [comment]
Lesson contents:

Screenshot_2020-02-28 Curriculum Materials(1)

**Comment by rjlynch** [comment]
Noticed a minor bug while looking over the review app
Screenshot 2020-02-28 at 16 39 08

File: app/presenters/teachers/lesson_contents_presenter.rb [hunk 1]
```diff
@@ -22,6 +22,7 @@ def initialize(lesson, teacher)
       @contents = lesson
         .lesson_parts
         .each_with_object({}) { |lesson_part, hash| hash[lesson_part] = lesson_part.activity_for(teacher) }
+        .reject { |_, activity| activity.nil? }
```
Comment (rjlynch) [inline app/presenters/teachers/lesson_contents_presenter.rb]: I know not all lesson_parts have an activity at the minute but do you think we should fail if there's a lesson_part without an activity when we go live?
Maybe make this reject conditional on an environment variable `LESSON_CONTENT_WIP` or something? 🤷‍♂️ Or add a note why this is here?

Comment (peteryates) [reply app/presenters/teachers/lesson_contents_presenter.rb]: I was wondering that. I think that we should probably have a state machine governing every CCP (or maybe unit) and before publishing it it should make these checks to ensure every lesson part has at least one activity, every unit has a lesson etc.

**Comment by rjlynch** [comment]
> Looks good, but there's a minor ordering bug with the lesson_parts and I can't see the icons on heroku, though it seems like heroku has compiled them all
> 
> ```
> Running bash on ⬢ dfe-curricul-cmb-545-ad-xe90mx... up, run.5457 (Free)
> 
> ~ $ ls public/packs/media/images/activity-icons/
> card-sort-5ebe09115bf5b59fcc4745af34242f6f.png     guided-reading-25040b082a4d1660a1979e32b345146a.png  reading-6db2bb83dbdf62d72e799a951c551674.png
> construction-972b6c00e7ddfee48418cb6b2b312cd9.png  individual-04ef562e83c54f0ed680d8c03b945882.png      teacher-81783a63459b474db8d37fa6df569c2f.png
> do-it-now-dcdebe1a7fe5b317fb1503ea71a2d1f7.png     mind-map-104999077b5395f5efa503428228952d.png        teacher-instruction-ae6882c5c513967a684f052f0e9942ca.png
> drawing-395e9596a069fc894a57225f18da3a13.png       pair-47d297f9636a0b5c8f43f9303adaca53.png            verbal-recap-c6b491c480b2280a7788eab0178d2d09.png
> feedback-604d6fb8c5c5f17e49e5f43ae9e9d405.png      plenary-85b02eee1b1fe663a180d7555ff8cec5.png         whole-9bf04fbc9c2d2bf6fd775c641357284a.png
> formative-5aff030ba50c9dd775dae6159324fe6d.png     poster-9eb87da8f0945a77e5285912f228a804.png          worksheet-22ffaac344a484e65799510525b060ae.png
> group-d71e6d2ad9ac3728c500104bb53ad6ae.png         practical-404cb6459c3accc7388957741e03e6ac.png       writing-3066252b5999aa3e9085fe5d177bd575.png
> 
> ~ $ ls app/webpacker/images/activity-icons/
> card-sort.png     drawing.png    group.png           mind-map.png  poster.png     teacher-instruction.png  whole.png
> construction.png  feedback.png   guided-reading.png  pair.png      practical.png  teacher.png              worksheet.png
> do-it-now.png     formative.png  individual.png      plenary.png   reading.png    verbal-recap.png         writing.png
> ```

This was just an issue with the review app not reseeding after further changes were pushed

**Pull request merged**